templates/base.html

```diff
@@ -77,6 +77,14 @@
 	<script src="{% static 'js/polyfill-find.js' %}"></script>
 	<script src="{% static 'js/focus-visible-polyfill.min.js' %}"></script>
 	<script src="{% static 'js/aplus.js' %}"></script>
+    <script>
+        function changeLanguage(lang) {
+            let baseUrl = "{{ BASE_URL }}";
+            let currentPath = window.location.pathname;
+            let newUrl = `${baseUrl}${currentPath}?hl=${lang}`;
+            history.replaceState(null, '', newUrl);
+        }
+    </script>
```

Comment:
Hi! Thank you for the simplification idea. However, I am not sure about removing `history.replaceState(null, '', newUrl);`. I use it to prevent users from creating a long history stack by clicking the button. Otherwise, they won't be able to instantly use 'go back' button to go back to the actual previous page, but the page with the last language parameter.

Reply:
Mostly done.